### PR TITLE
fix: screenshot and mirror with 0 bounds

### PR DESCRIPTION
**Pull request by onikienko**
Current code throws an exception if use `0` for `.screenshot()` bounds or for `.mirror()` secondID

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -40,14 +40,14 @@ function screenshot(id, options = {}) {
 
   if (options.bounds) {
     if (typeof options.bounds !== 'object') {
-      throw new TypeError(`'bounds' must be a number`)
-    } else if (!options.bounds.x || typeof options.bounds.x !== 'number') {
```
Comment (codebytere) [inline index.js:44]: Let's replace this check as well as the others with `Object.hasOwnProperty` or `x in options.bounds` instead of deleting it.

**Review by codebytere — CHANGES_REQUESTED** [review]
Thanks for this! Just one request, and then i'd also like to add test(s) (e.g. to ensure that 0 works and doesn't throw anymore) to ensure correct behavior before we get this merged 😁

File: index.js (hunk 1 above)
Comment (onikienko) [reply index.js:44]: If honest, I do not see the reason in that check. `options.bounds` is defined and has type `object`.  You ensure this in lines 41 and 42. 
So if `x` in `options.bounds` is not defined, `options.bounds.x` will be `undefined`.  So you will have check like `typeof undefined !== number`.  It's `true` and we will throw an error in next line as expected.
I can't imagine a scenario where we may need this additional check you are asking to add.

**Comment by onikienko** [comment]
Added tests. One of them to check `options.bounds` against `undefined`. To demonstrate that we do not need more checks here. 
Let me know what do you think.

**Review by codebytere — APPROVED** [review]
fair point - on further thought this is sufficient.

**Comment by onikienko** [comment]
Hi @codebytere 

Do you plan to publish a new release with this fix?

**Pull request merged**